resolver/addressable_resolver_test.go

```diff
@@ -532,13 +534,28 @@ func TestGetURIDestinationV1(t *testing.T) {
 		}, "notFound k8s service": {
 			dest:    duckv1.Destination{Ref: k8sServiceRef()},
 			wantErr: fmt.Sprintf("services %q not found", addressableName),
+		}, "with sample resolver": {
```

Comment:
I would like to see a test with two custom resolvers injected, and the first one passes.

Reply:
done and added a bonus test